deep_speech_2/model_utils/network.py

```diff
@@ -248,9 +269,16 @@ def deep_speech_v2_network(audio_data,
         stride_y=1,
         block_x=1,
         block_y=conv_group_height)
+    # remove padding part
+    remove_padding = paddle.layer.sub_seq(
```

Comment:
A better name would be like ```padding_removed_data``` or something else

Reply:
Done.